apps/user_ldap/lib/User_Proxy.php

```diff
@@ -400,15 +402,18 @@ public function createUser($username, $password) {
 	}
 
 	public function isUserEnabled(string $uid, callable $queryDatabaseValue): bool {
-		return $this->handleRequest($uid, 'isUserEnabled', [$uid, $queryDatabaseValue]);
+		if (!$this->treatRemnantsAsDisabled() && $this->deletedUsersIndex->isUserMarked($uid)) {
```

Comment:
```suggestion
		if ($this->treatRemnantsAsDisabled() && $this->deletedUsersIndex->isUserMarked($uid)) {
```

I think this is reversed?